router/router.nginx.conf.in

```diff
@@ -49,6 +49,7 @@ server {
 
 server {
     server_name batch.@domain@;
+    client_max_body_size 100m;
```

Comment:
I think @cseed said this needs to be changed in multiple places. Also I'm not sure 100m is going to be enough in the long term if Konrad wants to submit a batch with 12 million jobs that are each around 3.5KB. I think if this fixes it in the short term we should go ahead and do it.

Reply:
oh I see it's above as well.

Reply:
This changes it in the router as well as the gateway. The network path for production batch is this: `gateway` (hail.is) -> `router` (batch.@domain@) -> `batch`